docs/ui-components/price-breakdown.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Price Breakdown
+parent: UI Components
+nav_order: 4
+---
+
+# Price Breakdown
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+The price breakdown component displays information about Afterpay instalments and handles a number of common configurations.
+
+A total payment amount (represented as a `BigDecimal`) must be programatically set on the component to display Afterpay instalment information.
+
+```kotlin
+val totalAmount: BigDecimal = getTotalAmount()
+
+val paymentBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.priceBreakdown)
+paymentBreakdown.totalAmount = totalAmount
+```
+
+When the breakdown component is assigned a total amount that is valid for the merchant account, the 4 instalment amounts will be displayed.
+
+![Price breakdown Afterpay instalments are available][breakdown-available]
+
+When the total amount is not within the minimum and maximum payment values for the merchant account, the amounts available for Afterpay will be shown in the component.
+
+![Price breakdown Afterpay instalments are unavailable][breakdown-unavailable-min-max]
+
+When no minimum amount is set and the total amount is greater than the maximum payment values for the merchant account, the maximum amount available for Afterpay will be shown in the component.
+
+![Price breakdown Afterpay instalments are unavailable][breakdown-unavailable-max]
+
+When no payment amount has been set or the merchant account configuration has not been applied to the SDK, the component will default to a message stating Afterpay is available.
```

Comment:
If there is no amount and no configuration, does the default message state that Afterpay is available or unavailable?

```suggestion
When no payment amount has been set or the merchant account configuration has not been applied to the SDK, the component will default to a message stating Afterpay is unavailable.
```